flow-decomposition/src/main/java/com/powsybl/flow_decomposition/FlowDecompositionComputer.java

```diff
@@ -47,6 +50,11 @@ public FlowDecompositionComputer(FlowDecompositionParameters flowDecompositionPa
                                      String loadFlowProvider, String sensitivityAnalysisProvider) {
         this.parameters = flowDecompositionParameters;
         this.loadFlowParameters = loadFlowParameters;
+        if (!LoadFlowParameters.ConnectedComponentMode.MAIN.equals(loadFlowParameters.getConnectedComponentMode())) {
```

Comment:
Could you extract a constant for `LoadFlowParameters.ConnectedComponentMode.MAIN` please ?